src/components/FileInput/index.vue

```diff
@@ -74,27 +74,70 @@
     <div
       v-if="shouldDisplayDropbox"
       ref="dropbox"
-      :class="['drop-zone', isDragOver ? 'highlight' : null]"
+      :class="[
+        'drop-zone',
+        isDragOver ? 'highlight' : null,
+        { 'cancel-hover': selectedTabId === 'url' },
```

Comment:
it's a bit not common to start class name with verb. Let's rename `cancel-hover` to `hover-disabled` or `no-hover` or something similar